.github/workflows/cd.yml

```diff
@@ -155,6 +126,49 @@ jobs:
         run: |
           npx lerna publish from-package --yes
 
+      - name: Install Mustache
+        run: curl -sSL https://git.io/get-mo -o mo && chmod +x mo && sudo mv mo /usr/local/bin/
+
+      - name: generate templates
+        id: zip-template
+        run: .github/scripts/template-zip-autogen.sh
+
+      - name: release templates' alpha version to github
+        if: ${{ steps.zip-template.outputs.VERSION != '0' && github.event.inputs.preid != 'stable' }}
+        uses: marvinpinto/action-automatic-releases@latest
+        with:
+          repo_token: ${{ secrets.CD_PAT }}
+          prerelease: true
+          automatic_release_tag: steps.zip-template.outputs.VERSION
+          files: |
+            ./*.zip
+
+      - name: Create Templates Stable Release
+        if: ${{ steps.zip-template.outputs.VERSION != '0' && github.event.inputs.preid == 'stable' }}
```

Comment:
how does this action get version?

Reply:
add package in templates folder, lerna will manage to bump up templates version.

Reply:
I believe so. But, for this action, how does it know the release tag for templates? According to the doc, `tag: An optional tag for the release. If this is omitted the git ref will be used (if it is a tag)`, if the tag parameter is omitted, it uses git ref as tag. Does this workflow has that ref for it?

Reply:
template version will use steps.version-change.outputs.CHANGED template as the true version
these varibles work success on https://github.com/wenytang-ms-123/testavc/actions/workflows/cd.yml